content/en/infrastructure/containers/container_images.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Container Images
+kind: documentation
+further_reading:
+- link: "/security/vulnerabilities"
+  tag: "Documentation"
+  text: "Cloud Security Management Vulnerabilities"
+- link: "/security/vulnerabilities/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting Cloud Security Management Vulnerabilities"
+- link: "/security/cloud_security_management/setup/csm_pro?tab=aws#configure-the-agent-for-hosts-and-containers"
+  tag: "Documentation"
+  text: "Setting up CSM container vulnerabilities"
+- link: "/security/cloud_security_management/setup/csm_enterprise?tab=aws#configure-the-agent-for-vulnerabilities"
+  tag: "Documentation"
+  text: "Setting up CSM host vulnerabilities"
+- link: "https://www.datadoghq.com/blog/datadog-container-image-view/"
+  tag: "Blog"
+  text: "Enhance your troubleshooting workflow with Container Images in Datadog Container Monitoring"
```

Comment:
Should we put this as first external link since it's closely related to the new page?